global-news-app/src/components/Button/button.styled.tsx

```diff
@@ -0,0 +1,50 @@
+import MUIButton from "@mui/material/Button";
```

Comment:
i would change the file name to the `styles.ts`. i think it communicates the purpose effectively and without redundancy.